scripts/gen-ghci.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+######################################################################
+# Generates a .ghci config file for cardano-wallet.
+# Use this if you want faster reloads across packages.
+# Both this script and ghci need to be run from the nix-shell.
+######################################################################
+
+set -euo pipefail
+
+. "$(dirname "$0")/cabal-lib.sh"
+
+######################################################################
+# Config file generation
+
+make_ghci() {
+  echo "-- Generated by $0 for version $(get_cabal_version)"
+  echo "-- $(date)"
+  echo
+
+  ghci_flags | sed -e 's/^/:set /'
+
+  cat <<EOF
+
+:set prompt "λ "
+:set -package pretty-simple
```

Comment:
Nice! I think this means that we can remove `pretty-simple` as a dependency from the .cabal file?